src/soundio/sounddevicenetwork.cpp

```diff
@@ -445,7 +445,7 @@ void SoundDeviceNetwork::callbackProcessClkRef() {
         // This disables the denormals calculations, to avoid a
         // performance penalty of ~20
         // https://github.com/mixxxdj/mixxx/issues/7747
-#ifdef __SSE__
+#if defined(__SSE__) && !defined(__EMSCRIPTEN__)
```

Comment:
Yeah, it's available but since the denormals mode is fixed, we cannot use the set functions, so there is not really a point in having these blocks execute. The flush zero mode is fixed too, but in this case to being off, which isn't the mode we want. `_mm_setcsr` isn't available either, so we can't use the fallbacks from [denormalsarezero.h](https://github.com/mixxxdj/mixxx/pull/12917/files#diff-32be20b54e6491b218a1618b3b91e2d4376535535071e360fd5c48b9397a4ca5).

> Can you check that and adjust the debug message accordingly?

Do mean emitting a `qDebug()` message when targeting `__EMSCRIPTEN__`? Or only cond-compiling out the actual `_MM_SET` invocations and emitting a debug message where the fixed mode doesn't match our desired mode?

Reply:
Ok, than just add a comment about the situation. My original concern was that this is a implementation defined thing. 
The background is, that denormals are so slow, that the EQs and other filters are not usable. I am curious if they are usable with emscripten.

Reply:
Haven't gotten around to test EQ and filters in Mixxx yet (there are still some more foundational issues to be resolved, including audio support, before we can actually test that), but came across this interesting discussion upstream: https://github.com/WebAssembly/design/issues/148

Looks like flush-to-zero (FTZ)/denormals-as-zero (DAZ) were deliberately avoided in favor of IEEE-compliance, though they've hinted at adding it as an opt-in. Haven't found a specific proposal yet though.